package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "metal-myComponent",
+	"version": "0.0.0",
+	"description": "My awesome Metal.js project",
+	"license": "BSD",
+	"repository": "zxdgoal/metal-myComponent",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "src/MyComponent.js",
```

Comment:
This should point to `lib/Layout.js` and you should also have a `jsnext:main` field that will point to `src/Layout.js`, just like in [metal-modal's package.json](https://github.com/metal/metal-modal/blob/master/package.json#L15).